plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotClient.java

```diff
@@ -115,11 +119,12 @@
     private static final String TABLE_SCHEMA_API_TEMPLATE = "tables/%s/schema";
     private static final String ROUTING_TABLE_API_TEMPLATE = "debug/routingTable/%s";
     private static final String TIME_BOUNDARY_API_TEMPLATE = "debug/timeBoundary/%s";
-    private static final String QUERY_URL_TEMPLATE = "http://%s/query/sql";
+    private static final String QUERY_URL_TEMPLATE = "query/sql";
 
     private final List<URI> controllerUrls;
     private final HttpClient httpClient;
     private final PinotHostMapper pinotHostMapper;
+    private final boolean secureConnection;
```

Comment:
Please define `String scheme` instead and remove `secureConnection ? HTTPS_SCHEME : HTTP_SCHEME`.